paper/paper.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Vector: creating and manipulating jagged arrays of Lorentz vectors"
+tags:
+  - Python
+  - vector algebra
+  - high energy physics
+authors:
+  - name: Henry Schreiner
+    orcid: 0000-0002-7833-783X
+    equal-contrib: true
+    affiliation: 1
+  - name: Jim Pivarski
+    orcid: 0000-0002-6649-343X
+    equal-contrib: true
+    corresponding: true
+    affiliation: 1
+  - name: Saransh Chopra
+    orcid: 0000-0003-3046-7675
+    equal-contrib: true
+    affiliation: 1
+
+affiliations:
+  - name: Princeton University
+    index: 1
+date: 16 August 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+Vector algebra is a crucial component of data analysis pipelines in high energy
+physics, enabling physicists to transform raw data into meaningful results that
+can be visualized. Given that high energy physics data is not uniform, the
+vector algebra frameworks or libraries are expected to work readily on
+non-uniform or jagged data, allowing users to perform operations on an entire
```

Comment:
A definition of "jagged" will be needed. Ever since I started using this word, I've found that "ragged" is more common. A potentially confusing thing is that sometimes a "vector" is a collection-type data structure, and what we have here is a collection that contains (non-collection) vectors.